wp-content/themes/core/components/statistic/Controller.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tribe\Project\Templates\Components\statistic;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+
+/**
+ * Class Statistic
+ */
+class Controller extends Abstract_Controller {
+	/**
+	 * @var string
+	 */
+	public $tag;
```

Comment:
As of this morning, we're making all our properties `private` by default and using `get_*` methods to get the values in the template. We are going to do a sweep to update these so don't worry about hitting them all right now. Let's just get this one done!